docs/src/modules/spring/partials/entity-keys.adoc

```diff
@@ -0,0 +1,27 @@
+
+== Identifying the Entity
```

Comment:
In this partial, I tried to be generic enough. It's used in two different pages that use different examples (ShoppingCart for ES and Counter for VE). 

Because of that, the partial does not have any snippet and tries to explain everything inline (in text).